### DISCUSSION TITLE
Add support for twitter label meta

### DISCUSSION THREAD

**Discussion by pveyes** [vercel/next.js#46603] [Ideas]
### Describe the feature you'd like to request

Add new twitter meta fields to support custom label. Slack (not sure if other services do) reads `twitter:label` and `twitter:data` meta 

```html
<meta name="twitter:label1" content="Written by" />
<meta name="twitter:data1" content="Fatih Kalifa" />
<meta name="twitter:label2" content="reading Time" />
<meta name="twitter:data2" content="12 mins read" />
```

This will be rendered like this
<img width="556" alt="image" src="https://user-images.githubusercontent.com/1614415/222043808-e486f5e5-b0f1-40d6-bd3c-3b5599ae0b38.png">

Other examples: https://twitter.com/levelsio/status/828892835172671489?lang=en

### Describe the solution you'd like

Provide optional `twitter:labels` array in `Metadata` type

```ts
function generateMetadata() {
  return {
    twitter: {
      labels: [
        { key: "Written by", value: "Fatih Kalifa" },
        { key: "reading Time", value: "12 mins read" },
      ]
    }
  }
}
```

### Describe alternatives you've considered

Tried rendering the meta tag in `page.js`, but it didn't work

**Comment by ciruz** [comment]
You can define other metatags as well, like so:

```tsx
export const metadata = {
  title: "Home",
  description: "Welcome to Next.js",
  other: {
    ["twitter:label1"]: "written by",
    ["twitter:data1"]: "foobar",
  },
};
```

HTML result:
![image](https://user-images.githubusercontent.com/987947/222098960-58cd0d87-4989-49e9-99e4-9b148c380c0f.png)

So it's already possible.

**Reply by huozhi** [reply]
Yeah, should use `metadata.other` should be the one to use, `twitter:card` and `twitter:label` are not official properties defined in https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup.
